vision/google/cloud/vision/_gax.py

```diff
@@ -85,7 +83,7 @@ def _to_gapic_image(image):
               :class:`~google.cloud.vision.image.Image`.
     """
     if image.content is not None:
-        return image_annotator_pb2.Image(content=_to_bytes(image.content))
+        return image_annotator_pb2.Image(content=image.content)
```

Comment:
Because it's handled on the constructor of [`Image`](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2922/files#diff-fd4c8b0a5b301b229c829279de2c008bR57).